private/buf/buffetch/internal/module_ref.go

```diff
@@ -46,15 +46,15 @@ func newModuleRef(
 	if strings.Contains(path, "://") {
 		return nil, NewInvalidPathError(format, path)
 	}
-	moduleRef, err := bufmodule.ParseModuleRef(path)
+	moduleRef, err := bufparse.ParseRef(path)
 	if err != nil {
 		// TODO: this is dumb
 		return nil, NewInvalidPathError(format, path)
```

Comment:
`bufparse.ParseRef` only returns an error from the util `parseRefComponents`, which only ever returns a `ParseError`. To keep it consistent for now, we should continuing to return the invalid path error, but in the future, we should consider returning `ParseError` where appropriate from `buffetch`.